site2/docs/deploy-bare-metal.md

```diff
@@ -7,9 +7,9 @@ sidebar_label: Bare metal
 
 > ### Tips
 >
-> 1. Single-cluster Pulsar installations should be sufficient for all but the most ambitious use cases. If you are interested in experimenting with
-> Pulsar or using Pulsar in a startup or on a single team, it is simplest to opt for a single cluster. If you do need to run a multi-cluster Pulsar instance,
-> see the guide [here](deploy-bare-metal-multi-cluster.md).
+> 1. Single-cluster Pulsar installation should be sufficient for most use cases. If you are interested in experimenting with
+> Pulsar or using Pulsar in a startup or on a single team, it is simplest to opt for a single cluster. If you need to run a multi-cluster Pulsar instance,
+> see the [guide](deploy-bare-metal-multi-cluster.md).
 >
 > 2. If you want to use all builtin [Pulsar IO](io-overview.md) connectors in your Pulsar deployment, you need to download `apache-pulsar-io-connectors`
 > package and install `apache-pulsar-io-connectors` under `connectors` directory in the pulsar directory on every broker node or on every function-worker node if you
```

Comment:
I suggest we use `pulsar` directory or Pulsar directory. If it's not in code, we'd better write P in Pulsar in uppercase.